modules/jaeger-config-sampling.adoc

```diff
@@ -41,10 +37,11 @@ Jaeger libraries support the following samplers:
 |
 
 |sampling:
-  type:
+  options:
+    type:
 |Sampling strategy to use. (See descriptions above.)
-|Valid values are `const`, `probabilistic`, `ratelimiting`, and `remote`.
-|`remote`
+|Valid values are `probabilistic`, and `ratelimiting`.
+|`probabilistic`
 
 |sampling:
   options:
```

Comment:
options should
![image](https://user-images.githubusercontent.com/1004403/113330013-eb010c00-933b-11eb-894c-4d5014b32a9f.png)

The format mentioned above is wrong. Three rows are not required.
we can have one row as follows,
```yaml
spec:
  sampling:
    options:
      default_strategy: {}
      service_strategies: {}
```
for the `Values` table cell, you can get a text from https://www.jaegertracing.io/docs/1.22/sampling/#collector-sampling-configuration and 
https://github.com/jaegertracing/jaeger/blob/5062366157fff65a052dc6ac6779e7621054248d/plugin/sampling/strategystore/static/strategy.go#L38~L41

for `Default value` cell, we can list as follows,
```yaml
spec:
  sampling:
    options:
      default_strategy:
        type: probabilistic
        param: 1
```

Reply:
*laughs*  I know three rows aren't required, and that wasn't what I was trying to imply in the table.  But one of your previous comments was "In my opinion, to avoid confusion always start from the root element, spec" but maybe you just meant in the code examples, not the table?

Reply:
Or, reading your other comment, just the first row in the table?

Reply:
@JStickler The first row is enough and should be updated as I mentioned.

```yaml
spec:
  sampling:
    options:
      default_strategy: {}
      service_strategies: {}
```    


The other two rows can be removed also those are wrong.
under the sampling `options`,
* we should have only these two keys(strategies),
* `default_strategy` and `service_strategies`.


the options you mentioned in row#2 (`type`) and row#3 (`type`, `param`), should come under a strategy(`default_strategy`, `service_strategies`), those `type` and `param` can not be directly under the `options`.